app/src/main/java/org/xbmc/kore/service/NotificationObserver.java

```diff
@@ -224,51 +230,83 @@ private void notifyPlaying(PlayerType.GetActivePlayersReturnType getActivePlayer
+            rewindPendingIntent = buildActionPendingIntent(getActivePlayerResult.playerid,
+                    IntentActionsService.ACTION_REWIND);
             rewindIcon = R.drawable.ic_fast_rewind_white_24dp;
-            ffPendingIntent = buildActionPendingIntent(getActivePlayerResult.playerid, IntentActionsService.ACTION_FAST_FORWARD);
+            ffPendingIntent = buildActionPendingIntent(getActivePlayerResult.playerid,
+                    IntentActionsService.ACTION_FAST_FORWARD);
             ffIcon = R.drawable.ic_fast_forward_white_24dp;
         }
 
+        jumpBackPendingIntent = buildActionPendingIntent(getActivePlayerResult.playerid,
+                IntentActionsService.ACTION_JUMP_BACKWARD);
+        jumpBackIcon = R.drawable.ic_skip_backward_white_24dp;
+
+        jumpForwardPendingIntent = buildActionPendingIntent(getActivePlayerResult.playerid,
+                IntentActionsService.ACTION_JUMP_FORWARD);
+        jumpForwardIcon = R.drawable.ic_skip_forward_white_24dp;
+
         final NotificationCompat.Builder builder =
-            new NotificationCompat.Builder(service, NOTIFICATION_CHANNEL)
-                .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
+                new NotificationCompat.Builder(service, NOTIFICATION_CHANNEL);
+
+        builder.setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
                 .setSmallIcon(smallIcon)
                 .setShowWhen(false)
                 .setOngoing(true)
-                .addAction(rewindIcon, service.getString(R.string.rewind), rewindPendingIntent) // #0
-                .addAction(playPauseIcon, service.getString(R.string.play), playPausePendingIntent)  // #1
-                .addAction(ffIcon, service.getString(R.string.fast_forward), ffPendingIntent)     // #2
-                .setStyle(new androidx.media.app.NotificationCompat.MediaStyle()
-                              .setShowActionsInCompactView(0, 1, 2))
                 .setContentIntent(remoteStartPendingIntent)
                 .setContentTitle(title)
                 .setContentText(underTitle);
 
+        if (getItemResult.type.equals(ListType.ItemsAll.TYPE_EPISODE)
+                || getItemResult.type.equals(ListType.ItemsAll.TYPE_MOVIE)
+        ) {
+            builder.addAction(rewindIcon, service.getString(R.string.rewind), rewindPendingIntent) // #0
+                    .addAction(jumpBackIcon, service.getString(R.string.rewind), jumpBackPendingIntent) // #1
+                    .addAction(playPauseIcon, service.getString(R.string.play), playPausePendingIntent)  // #2
+                    .addAction(jumpForwardIcon, service.getString(R.string.fast_forward),
+                            jumpForwardPendingIntent) // #3
+                    .addAction(ffIcon, service.getString(R.string.fast_forward), ffPendingIntent)   // #4
+                    .setStyle(new androidx.media.app.NotificationCompat.MediaStyle()
+                            .setShowActionsInCompactView(1, 2, 3));
```

Comment:
The order of the buttons should be 
`jumpBack, rewind, play, ffwd, jumpFwd`
and the buttons shown on the compact view should be the 
`rewind, play, ffwd`
as that is the current default.